modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -109,38 +150,106 @@ public void Dispose()
 
         // IList
 
+        /// <summary>
+        /// This will always be false. It's needed by <see cref="IList"/>.
+        /// </summary>
         public bool IsReadOnly => false;
```

Comment:
If we don't actually use these `IList` properties we could implement them explicitly.

Reply:
What do you mean?

Reply:
```suggestion
        bool IList.IsReadOnly => false;
```

Implemented like this it's like the property is private and the users won't be able to use it unless they cast to `IList` so if we don't actually want the property to show up while still implementing the interface.